tests/field/test_magneticfields.py

```diff
@@ -1005,6 +1007,45 @@ def test_to_vtk(self):
         bs = BiotSavart(coils)
         bs.to_vtk('/tmp/bfield')
 
+    def test_to_mgrid(self):
+        curves, currents, ma = get_ncsx_data()
+        nfp = 3
+        coils = coils_via_symmetries(curves, currents, nfp, True)
+        bs = BiotSavart(coils)
+        with tempfile.TemporaryDirectory() as tmpdir:
+            filename = Path(tmpdir) / "mgrid.bfield.nc"
+            bs.to_mgrid(filename, nfp=nfp)
+
+            # Compare the B data in the file to a separate evaluation here
+            f = netcdf_file(filename, mmap=False)
```

Comment:
Better to use context manager as a good coding style

Reply:
Done.